### ISSUE TITLE
Incorrect UNRAID instructions in the WiKi

### ISSUE THREAD

**Issue by HVR88** [gaseous-project/gaseous-server#361]
UNRAID instructions in the wiki...

> In Main -> BootDevice -> config -> plugins -> dockerMan -> templates-user click UPLOAD (at the bottom) then select the gaseous.xml files

The above is not a thing in unraid. There's no upload facility in the UNRAID webUI when browsing MAIN nor BootDevice. You can browse the boot device filesystem, but again, there's no way to upload to it from the UI.

Just put the container up on Unraid's CA store or at the very least, DockerHub where most other people obtain their containers.

**Comment by michael-j-green** [comment]
Hi, I do not have an Unraid server to test these instructions with, they were provided by other users on our Discord server.

The container for Gaseous is already available on Dockerhub, and is referenced in the supplied docker-compose.yml file.

**Comment by cansas** [comment]
In the Main section of the Unraid dashboard, click on the box arrow to the left of the boot device name and you will get a filesystem to navigate and upload.

![Screenshot 2024-05-30 at 8 43 35 AM](https://github.com/gaseous-project/gaseous-server/assets/5727492/ced7fb90-9b16-4ddf-816d-0bca7c6db663)

**Comment by HVR88** [comment]
I've been running Unraid for about 10 years and I love to learn something new. I've never seen/heard of any way to upload or modify the filesystem using the Unraid config UI.

The browse icon to the left of the Flash drive (and every drive and share on the array) shows the contents of that drive/share, but I don't know any way to modify or upload via that UI. It's hierarchical file listing, but static otherwise.

**Comment by HVR88** [comment]
What do you have enabled to give you the file operations above?

This is how the bottom of any browse appears by default:

![image](https://github.com/gaseous-project/gaseous-server/assets/33399216/fbed2591-b786-4b29-b86c-f785d14d41e5)

**Comment by HVR88** [comment]
Very handy add-on too, so I've installed it. :)

**Comment by Tom1243** [comment]
Yeah thats my bad. This was already brought up in the past. Still needs to be changed :P  to include that.

**Comment by michael-j-green** [comment]
Duplicate of #117 

Leaving open for tracking purposes
